charts/ia-homeoffice-integration-api/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: ia-homeoffice-integration-service
+home: https://github.com/divisathyan/ia-ho-integration-services
```

Comment:
please make sure this is pointing to the hmcts repo

Reply:
corrected